test/user.test.js

```diff
@@ -294,17 +300,15 @@ describe('User', function() {
         },
         function(next) {
           User.login({email: 'b@c.com', password: 'bar'}, function(err, accessToken) {
-            accessTokenId = accessToken.userId;
             if (err) return next(err);
-            assert(accessTokenId);
+            expect(accessToken.toJSON()).to.have.property('userId', userIds[0]);
```

Comment:
Perhaps you could change the signature of `assertGoodToken` to accept `userId` instead of a full user instance, and then use this helper here, instead of spelling out the assertion again and again?

Same comment for the `userIds[1]` check below.